src/components/SwitchTheme.tsx

```diff
@@ -6,6 +6,8 @@ const SwitchTheme: React.FC = () => {
   const themeContext = useContext<ThemeContextT>(ThemeContext);
   const { theme, setTheme } = themeContext;
 
+  // esto es una prueba de commit 
```

Comment:
We should always use English when commenting the code. But other than that, good job on your first Pull Request! Welcome to the team :)

Reply:
Make the commit to explain some code